crates/sui-cost/src/estimator.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2022, Mys&ten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+use std::fs;
+use std::str::FromStr;
+use strum_macros::Display;
+use strum_macros::EnumString;
+use sui_types::gas::GasCostSummary;
+use sui_types::messages::SingleTransactionKind;
+use sui_types::messages::TransactionKind;
+
+pub const ESTIMATE_FILE: &str = "data/estimate.toml";
+#[derive(
+    Debug, Eq, PartialEq, Hash, Serialize, Deserialize, Ord, PartialOrd, Clone, Display, EnumString,
+)]
+pub enum CommonTransactionCosts {
+    Publish,
+    MergeCoin,
+    SplitCoin(usize),
+    TransferWholeCoin,
+    TransferWholeSuiCoin,
+    TransferPortionSuiCoin,
+    SharedCounterCreate,
+    SharedCounterAssertValue,
+    SharedCounterIncrement,
+}
+
+pub fn estimate_computational_costs_for_transaction(
```

Comment:
here it returns all computation, storage and storage rebate, so `estimate_costs_for_transaction`?